docs/ingestion/index.md

```diff
@@ -737,3 +741,11 @@ The `indexSpec` object can include the following properties:
 
 Beyond these properties, each ingestion method has its own specific tuning properties. See the documentation for each
 [ingestion method](#ingestion-methods) for details.
+
+#### `appendableIndexSpec`
+
+|Field|Description|Default|
+|-----|-----------|-------|
+|type|Each in-memory index has its own tuning type code. You must specify the type code that matches your in-memory index. Common options are `onheap`, and `offheap`.|`onheap`|
+
+Beyond these properties, each in-memory index has its own specific tuning properties.
```

Comment:
Do you think if users want to change this config ever for now? I remember @gianm mentioned that the offheap incremental index has some performance issue, which is why we don't use it for indexing. If you think this knob is useful for users, please add more details how each index type is different and when it's recommended to use what. Otherwise, I would suggest not documenting this know at least for now.

Reply:
Actually, by @a2l007 suggestion, I deferred the support for offheap ingestion to a different PR.
So this documentation should not include the `offheap` option anyway.
I'll removed this documentation for now.